docs/number-input.md

```diff
@@ -0,0 +1,62 @@
+<link href="style.css" rel="stylesheet"></link>
+
+# PsNumberInput <a name="PsNumberInput"></a>
+
+`<ps-number-input>` allows the selection of a value or a range from a range via mouse, touch, or keyboard.
+
+---
+
+## API <a name="PsNumberInputApi"></a>
+
+### Import <a name="PsNumberInputImport"></a>
+
+```ts | js
+import { PsNumberInputModule } from '@prosoft/components/number-input';
+```
+
+---
+
+## PsNumberInputComponent <a name="PsNumberInputComponent"></a>
+
+### Properties <a name="PsNumberInputComponentProperties"></a>
+
+| Name                                   | Description                                                           |
+| -------------------------------------- | --------------------------------------------------------------------- |
+| `value: number`                        | The current value.                                                    |
+| `min: number`                          | Sets the lowest selectable number.                                    |
+| `max: number`                          | Sets the highest selectable number.                                   |
+| `stepSize: number`                     | Sets the step size of each click on the up/down buttons.              |
+| `decimals: number`                     | Sets the number of allowed decimals.                                  |
```

Comment:
fractions

Reply:
I talked with @Yirica and @6rayWolf and they like decimals more. I guess decimals is also ok for you because you approved, so I'll merge it now.